tsl/test/sql/recompress_chunk_segmentwise.sql

```diff
@@ -291,6 +291,23 @@ insert into nullseg_many values (:'start_time', 1, NULL, NULL);
 SELECT compress_chunk(:'chunk_to_compress');
 select * from :compressed_chunk_name;
 
+-- Test behaviour when no segmentby columns are present
+SET timescaledb.debug_compression_path_info TO ON;
```

Comment:
Lets enable this GUC for the complete test so we can verify that the correct index is being used for each recompression.